Userland/Games/MasterWord/main.cpp

```diff
@@ -4,8 +4,8 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
-#include "WordGame.h"
 #include "MainWidget.h"
+#include "WordGame.h"
```

Comment:
I think this commit should be squashed with the previous one

Reply:
Sure thing - is there a way for me to do that on my end? I think there is an option to "Squash Commits and Merge" if the rest of the code is ok to go in, but I'm also happy to close this PR and make a new one with just the one commit if that works better.

Reply:
Thanks, much appreciated! That should be done now, hopefully that did the trick

Reply:
Yep, that looks fine, you can mark this as resolved